CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.3.1-dev.2
+
+* Replaced uses of deprecated `asExternalTypedData` with `asTypedList`.
```

Comment:
Does this change the min. Dart SDK requirements for this package? If so, we should update the pubspec as well.

Reply:
I don't think `asTypedList` has been published in a dev release yet, but we needed to make the change to ship D26, since this package is used in the SDK and we want to remove deprecated methods before shipping.

After shipping we should update the minimum constraint to D26 /cc @dcharkes @mkustermann